common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala

```diff
@@ -75,7 +75,8 @@ object Mappings {
     "subLocation" -> standardAnalysedString,
     "city" -> standardAnalysedString,
     "state" -> standardAnalysedString,
-    "country" -> standardAnalysedString
+    "country" -> standardAnalysedString,
+    "englishAnalysedCatchAll" -> sStemmerAnalysedString
```

Comment:
good call, if we get labels in there it'll fix this too (i'm pretty sure >.<): https://github.com/guardian/grid/issues/331